sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/Combine.java

```diff
@@ -1447,6 +1504,11 @@ public boolean getInsertDefault() {
     public GlobalCombineFn<? super InputT, ?, OutputT> getCombineFn() {
       return fn;
     }
+
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      Combine.populateDisplayData(builder, fn, fnClass);
```

Comment:
`insertDefault` is an internal name-- any thoughts on a good name for the display data? At this point I have `emitDefaultOnEmptyInput`, based on this documentation:

    /**
     * Returns a {@link PTransform} identical to this, but that does not attempt to
     * provide a default value in the case of empty input.  Required when the input
     * is not globally windowed and the output is not being used as a side input.
     */
    public Globally<InputT, OutputT> withoutDefaults() {